Core/Date.cs

```diff
@@ -30,315 +30,146 @@ namespace System
-		}
+		public static bool operator !=(Date d1, Date d2) => d1._dt != d2._dt;
 
-		public static Date operator +(Date d, TimeSpan t)
-		{
-			return new Date(d._dt + t);
-		}
+		public static Date operator +(Date d, TimeSpan t) => new Date(d._dt + t);
 
-		public static bool operator <(Date d1, Date d2)
-		{
-			return d1._dt < d2._dt;
-		}
+		public static bool operator <(Date d1, Date d2) => d1._dt < d2._dt;
 
-		public static bool operator <=(Date d1, Date d2)
-		{
-			return d1._dt <= d2._dt;
-		}
+		public static bool operator <=(Date d1, Date d2) => d1._dt <= d2._dt;
 
-		public static bool operator ==(Date d1, Date d2)
-		{
-			return d1._dt == d2._dt;
-		}
+		public static bool operator ==(Date d1, Date d2) => d1._dt == d2._dt;
 
-		public static bool operator >(Date d1, Date d2)
-		{
-			return d1._dt > d2._dt;
-		}
+		public static bool operator >(Date d1, Date d2) => d1._dt > d2._dt;
 
-		public static bool operator >=(Date d1, Date d2)
-		{
-			return d1._dt >= d2._dt;
-		}
+		public static bool operator >=(Date d1, Date d2) => d1._dt >= d2._dt;
 
-		public static implicit operator DateTime(Date d)
-		{
-			return d._dt;
-		}
+		public static implicit operator DateTime(Date d) => d._dt;
 
-		public static explicit operator Date(DateTime d)
-		{
-			return new Date(d);
-		}
+		public static explicit operator Date(DateTime d) => new Date(d);
 
-		public int Day
-		{
-			get
-			{
-				return _dt.Day;
-			}
-		}
+		public int Day => _dt.Day;
 
-		public DayOfWeek DayOfWeek
-		{
-			get
-			{
-				return _dt.DayOfWeek;
-			}
-		}
+		public DayOfWeek DayOfWeek => _dt.DayOfWeek;
 
-		public int DayOfYear
-		{
-			get
-			{
-				return _dt.DayOfYear;
-			}
-		}
+		public int DayOfYear => _dt.DayOfYear;
 
-		public int Month
-		{
-			get
-			{
-				return _dt.Month;
-			}
-		}
+		public int Month => _dt.Month;
 
-		public static Date Today
-		{
-			get
-			{
-				return new Date(DateTime.Today);
-			}
-		}
+		public static Date Today => new Date(DateTime.Today);
 
-		public int Year
-		{
-			get
-			{
-				return _dt.Year;
-			}
-		}
+		public int Year => _dt.Year;
 
-		public long Ticks
-		{
-			get
-			{
-				return _dt.Ticks;
-			}
-		}
+		public long Ticks => _dt.Ticks;
 
-		public Date AddDays(int value)
-		{
-			return new Date(_dt.AddDays(value));
-		}
+		public Date AddDays(int value) => new Date(_dt.AddDays(value));
 
-		public Date AddMonths(int value)
-		{
-			return new Date(_dt.AddMonths(value));
-		}
+		public Date AddMonths(int value) => new Date(_dt.AddMonths(value));
 
-		public Date AddYears(int value)
-		{
-			return new Date(_dt.AddYears(value));
-		}
+		public Date AddYears(int value) => new Date(_dt.AddYears(value));
 
-		public static int Compare(Date d1, Date d2)
-		{
-			return d1.CompareTo(d2);
-		}
+		public static int Compare(Date d1, Date d2) => d1.CompareTo(d2);
 
-		public int CompareTo(Date value)
-		{
-			return _dt.CompareTo(value._dt);
-		}
+		public int CompareTo(Date obj) => _dt.CompareTo(obj._dt);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Rename parameter 'obj' to 'other' to match the interface declaration.](https://app.codacy.com/manual/siriak/MoneyManager/pullRequest?prid=4931949)